phase2/kubeadm/configure-vm-kubeadm-node.sh

```diff
@@ -1,4 +1,10 @@
 
 # This is not meant to run on its own, but extends phase2/kubeadm/configure-vm-kubeadm.sh
 
-kubeadm join --token "$KUBEADM_TOKEN" "$KUBEADM_MASTER_IP:443" --skip-preflight-checks --discovery-token-unsafe-skip-ca-verification true
+OPTS=""
+if [[ $KUBEADM_VERSION == "stable" ]] || [[ $(semver_compare $KUBEADM_VERSION "v1.8.0") -gt 1 ]]; then
```

Comment:
@shashidharatd can we make sure the "stable" version is always greater than 1.8.0? I guess so as the current version found in the link is v1.8.4
https://github.com/kubernetes/kubernetes-anywhere/blob/7eee9cb5d0767619f1833fb5242d2543b04844dd/phase2/kubeadm/do#L39

dl.k8s.io/release/stable.txt

Reply:
@xiangpengzhao, as you pointed out, current stable version is v1.8.4 and it will always go up. while pushing the release to GCS there is check which ensures pointers like stable.txt and latest.txt are updated only if the newer version is greater than the existing version.